index.js

```diff
@@ -936,20 +936,34 @@ class GithubScm extends Scm {
      * @param  {String}   config.type      Can be 'pr' or 'repo'
      * @param  {Object}   config.payload   The webhook payload received from the SCM service.
      * @param  {String}   config.token     Service token to authenticate with Github
+     * @param  {String}   config.scmUri    The scmUri to get PR info of
+     * @param  {Integer}  config.prNum     The PR number
      * @return {Promise}                   Resolves to an array of filenames of the changed files
      */
-    async _getChangedFiles({ type, payload, token }) {
+    async _getChangedFiles({ type, payload, token, scmUri, prNum }) {
         if (type === 'pr') {
+            let owner;
+            let repo;
+            let number;
+
+            if (scmUri) {
```

Comment:
we should probably add a test to cover this if condition

Reply:
do you need to make sure both `scmUri` && `prNum` exist?

Reply:
👍